packages/page-staking/src/Nomination/TabsHeader.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+interface Props {
+  setCurrentStep: (id: string) => void;
+  stepsState: string[];
+  steps: string[];
+  currentStep: string;
+}
+
+function TabsHeader({ stepsState, setCurrentStep, steps, currentStep }: Props): React.ReactElement<Props> {
+
+  function setCurrentValue(event: React.MouseEvent, id: string) {
+    event.preventDefault();
+    setCurrentStep(id);
+  }
+
+  return (
+    <div className="ui ordered top attached steps">
+      <a onClick={event => setCurrentValue(event, steps[0])} className={`step ${stepsState[0]} ${currentStep === steps[0] ? 'active' : ''}`}>
+        <div className="content">
+          <div className="title">Account</div>
+        </div>
+      </a>
+      <a onClick={event => setCurrentValue(event, steps[1])} className={`step ${stepsState[1]} ${currentStep === steps[1] ? 'active' : ''}`}>
```

Comment:
Always do `(event) => ...` (with the wrapping `()`'s, but as above never inline. useCallback)